pkg/policy/api/rule_validation.go

```diff
@@ -199,3 +213,47 @@ func (cidr CIDR) Validate() error {
 
 	return nil
 }
+
+// Validate validates a CIDRRule by checking that the CIDR itself is valid,
+// and ensuring that all of the exception CIDRs are contained within the CIDR.
+func (c CIDRRule) Validate() error {
+	err := c.Cidr.Validate()
+	if err != nil {
+		return err
+	}
+
+	// Don't need to check error as this was done in validation of CIDR.
```

Comment:
Well, do you check it or not? ;-)

Reply:
I changed the logic to ensure that only proper CIDR notation is provided (<IP>/<prefix>) instead of accepting single IPs in `CIDRRule` validation.